src/main/java/edu/cornell/kfs/coa/businessobject/CuAccountGlobal.java

```diff
@@ -8,146 +8,326 @@
+		}
+
+		if (StringUtils.isNotBlank(accountsSupervisorySystemsIdentifier)) {
+		    account.setAccountsSupervisorySystemsIdentifier(accountsSupervisorySystemsIdentifier);
+		}
+
+		if (StringUtils.isNotBlank(accountManagerSystemIdentifier)) {
+		    account.setAccountManagerSystemIdentifier(accountManagerSystemIdentifier);
+		}
+
+		if (StringUtils.isNotBlank(organizationCode)) {
+		    account.setOrganizationCode(organizationCode);
+		}
+
+		if (StringUtils.isNotBlank(subFundGroupCode)) {
+		    account.setSubFundGroupCode(subFundGroupCode);
+		}
+
+		if (StringUtils.isNotBlank(accountCityName)) {
+		    account.setAccountCityName(accountCityName);
+		}
+
+		if (StringUtils.isNotBlank(accountStateCode)) {
+		    account.setAccountStateCode(accountStateCode);
+		}
+
+		if (StringUtils.isNotBlank(accountStreetAddress)) {
+		    account.setAccountStreetAddress(accountStreetAddress);
+		}
+
+		if (StringUtils.isNotBlank(accountZipCode)) {
+		    account.setAccountZipCode(accountZipCode);
+		}
+
+		if (accountExpirationDate != null) {
+		    account.setAccountExpirationDate(new Date(accountExpirationDate.getTime()));
+		}
+
+		if (StringUtils.isNotBlank(continuationFinChrtOfAcctCd)) {
+		    account.setContinuationFinChrtOfAcctCd(continuationFinChrtOfAcctCd);
+		}
+
+		if (StringUtils.isNotBlank(continuationAccountNumber)) {
+		    account.setContinuationAccountNumber(continuationAccountNumber);
+		}
+
+		if (StringUtils.isNotBlank(incomeStreamFinancialCoaCode)) {
+		    account.setIncomeStreamFinancialCoaCode(incomeStreamFinancialCoaCode);
+		}
+
+		if (StringUtils.isNotBlank(incomeStreamAccountNumber)) {
+		    account.setIncomeStreamAccountNumber(incomeStreamAccountNumber);
+		}
+
+		if (StringUtils.isNotBlank(accountCfdaNumber)) {
+		    account.setAccountCfdaNumber(accountCfdaNumber);
+		}
+
+		if (StringUtils.isNotBlank(financialHigherEdFunctionCd)) {
+		    account.setFinancialHigherEdFunctionCd(financialHigherEdFunctionCd);
+		}
+
+		if (StringUtils.isNotBlank(accountSufficientFundsCode)) {
+		    account.setAccountSufficientFundsCode(accountSufficientFundsCode);
+		}
+
+		if (StringUtils.isNotBlank(getLaborBenefitRateCategoryCode())) {
+		    account.setLaborBenefitRateCategoryCode(getLaborBenefitRateCategoryCode());
+		}
+
+		if (pendingAcctSufficientFundsIndicator != null) {
+		    account.setPendingAcctSufficientFundsIndicator(pendingAcctSufficientFundsIndicator);
+		}
+
+		if (StringUtils.isNotBlank(majorReportingCategoryCode)) {
+		    ((AccountExtendedAttribute) account.getExtension()).setMajorReportingCategoryCode(majorReportingCategoryCode);
+		}
+		
+		if (StringUtils.isNotBlank(accountPhysicalCampusCode)) {
+		    account.setAccountPhysicalCampusCode(accountPhysicalCampusCode);
+		}
+
+		if (ObjectUtils.isNotNull(accountEffectiveDate)) {
+		    account.setAccountEffectiveDate(accountEffectiveDate);
+		}
+		
+		if(accountOffCampusIndicator){
```

Comment:
As per my previous comment related to Booleans, you may be able to update many of these similar comparisons to check for non-null values instead, allowing for setting the properties to either true or false on a global update. Unless there's a specific business requirement to only allow them to be set to true on a global update.